src/core/action-creators.js

```diff
@@ -14,10 +14,11 @@ import * as Actions from './action-types';
  * @param {string} type - The move type.
  * @param {Array}  args - Additional arguments.
  * @param {string}  playerID - The ID of the player making this action.
+ * @param {string}  secret - (optional) The secret for the player making this action.
  */
-export const makeMove = (type, args, playerID) => ({
+export const makeMove = (type, args, playerID, secret) => ({
```

Comment:
Let's call this credentials instead of secret?

Reply:
In this particular case, or everywhere? If everywhere, sure, happy to do it. If just in the one case, it feels a little inconsistent.

Reply:
@nicolodavis In this particular case, or everywhere? If everywhere, sure, happy to do it. If just in the one case, it feels a little inconsistent.

Reply:
Just want to avoid confusion with the game secrets that we strip elsewhere for secret state in games. Changing it everywhere in this PR sounds good.